src/components/ControlDropdown/ControlDropdown.tsx

```diff
@@ -18,19 +17,26 @@ import { Icon } from '../Icon';
 import { H4 } from '../typography';
 import { ControlDropdownProps } from './ControlDropdown.types';
 import { ControlDropdownPlacements } from './ControlDropdown.enums';
+import ControlledDropdown from '../Dropdown/ControlledDropdown';
+import { Placements } from '../Dropdown/Dropdown.types';
 
 const CloseButton = styled(Button)`
   padding: 0;
   width: ${pxToRem(16)};
   height: ${pxToRem(16)};
 `;
 
+const placementAdapter: Record<typeof ControlDropdownPlacements, Placements> = {
```

Comment:
I was checking that this component is not used anywhere so feel free to remove the adapter and use the real `*-start` and `*-end` values. We probably can remove the restriction to just bottom position since the popper.js used behind the dropdowns will use this only as prefered default position but will reposition when collision should happen.